src/controllers/router-store/utils/path-utils.ts

```diff
@@ -28,3 +28,14 @@ export const getRelativePath = (path: Href | Location): string | Location => {
 
   return `${url.pathname}${url.query}${url.hash}`;
 };
+
+export const updateQueryParams = (location: Location, query: Query): string => {
+  // @ts-ignore stringify accepts two params but it's type doesn't say so
+  const stringifiedQuery = qs.stringify(query, true);
+
+  return `${location.pathname}${stringifiedQuery}${location.hash}`;
+};
+
+export const getURLFromLocation = (location: Location): string => {
```

Comment:
nit; can this be named `getRelativeURLFromLocation`?

Reply:
Yup